.github/workflows/test.yml

```
@@ -0,0 +1,23 @@
+name: test
+on: [push, pull_request]
+jobs:
+  test:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Should this be 3.7, since that's the Python data-importer uses?

---

I think the data importer version should get bumped up instead!

---

I think the data importer version should get bumped up instead!

---

Would it make any sense to matrix both versions for now, e.g. 
```suggestion
        python-version: [3.8, 3.7]
```

---

sounds good!!